radio/wifi/wifi.go

```diff
@@ -0,0 +1,180 @@
+package wifi
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os/exec"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/lair-framework/go-nmap"
+	"github.com/parnurzeal/gorequest"
+)
+
+type Host struct {
+	id  string
+	ip  string
+	mac string
+}
+
+func StartHotspot() error {
+	if err := activateConnection("resin-hotspot"); err != nil {
+		return err
+	}
+
+	// Give the wifi devices a chance to connect
+	time.Sleep(5 * time.Second)
+	return nil
+}
+
+func StopHotspot() error {
+	return activateConnection("resin-wifi")
```

Comment:
This is a little misleading, the function is named `StopHotspot` but the call is `activateConnection` so maybe add a comment specifying what is actually going on, or rename the function.